hack/update-k8s-deps.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export GO111MODULE=on
+export K8S_VERSION="${1:-v0.18.8}"
+
+K8S_DEPS=(
```

Comment:
You could use buoy to update k8s deps for you too, just use the same code that is in use in go_update_deps and change the domain to k8s.io

Reply:
Thanks @n3wscott, I didn't know about buoy. I'll create an issue to look into using this instead.

Reply:
https://github.com/knative-sandbox/async-component/issues/24